Description: Use the avconv executable instead of ffmpeg
 In Debian, ffmpeg has been deprecated in favor of  the avconv executable
 since the switch to libav upstream.
Forwarded: not-needed
Author: Simon Chopin <chopin.simon@gmail.com>
Last-Update: 2012-01-18
--- a/audioread/ffdec.py
+++ b/audioread/ffdec.py
@@ -35,7 +35,7 @@
     def __init__(self, filename):
         try:
             self.proc = subprocess.Popen(
-                ['ffmpeg', '-i', filename, '-f', 's16le', '-'],
+                ['avconv', '-i', filename, '-f', 's16le', '-'],
                 stdout=subprocess.PIPE, stderr=subprocess.PIPE
             )
         except OSError:
